src/System.Xml.XPath.XDocument/ref/System.Xml.XPath.XDocument.csproj

```diff
@@ -3,14 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Xml.XPath.XDocument.cs" />
     <Compile Include="System.Xml.XPath.XDocument.Manual.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
```

Comment:
I don't think this remove was intentional. Even though project would still work, project.json won't appear in VS anymore, so let's bring this back.